src/main/java/com/google/sps/servlets/ProfileBuilderServlet.java

```diff
@@ -0,0 +1,66 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import com.google.sps.data.ProfessionalProfile;
+import com.google.sps.data.StudentProfile;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/profile")
+public class ProfileBuilderServlet extends HttpServlet {
+
+  private void storeProfile(String userType, String fullname, String input2, String input3) {
+
```

Comment:
Maybe split storeProfile into storeStudentProfile and storeProfessionalProfile, then you can have more descriptive input names (I'd generally avoid having the same input, like 'input2' and 'input3', mean different things based on the other arguments. It makes it hard to understand what a function does at a glance)

Reply:
Done